README.md

```diff
@@ -1,17 +1,54 @@
-# Software Engineering B4 Project Repo
+# PyramidQuest
+## What is PyramidQuest
+PyramidQuest is a game that aims to teach children educational knowledge in a playful way. As a character you are an archaeologist who is in a pyramid and is looking for a way up. This pyramid consists of two mazes. In the labyrinths he encounters obstacles such as: puzzles when opening doors, as well as different types of enemies and traps. There are 3 stars in each of the two levels that you can collect to get a better score. There is also a timer that the child can use to measure their time.     
 
-### Projektmanagement:
-* Manuel
-* Leon
+### Built With
 
-### Vertriebsmanagement:
-* Amier
-* Lucas
+* [Unity]
 
-### Techmanagement:
-* Baran
-* Hannah
+## Getting Started
+To get a local copy up and running follow these simple steps.
 
-### Qualitätmanagement:
-* Nam
-* Ikram
+### Prerequisites
+
+You need a Windows 10 operating system or newer.
+
+### Installation
```

Comment:
Installation part needs Development instructions.

Installation instructions need to be from "visiting github" all the way to "execute PyramidQuest.exe"